src/markdown-preview-view/markdown-preview-view.ts

```diff
@@ -144,7 +146,7 @@ export abstract class MarkdownPreviewView {
             html,
             this.renderLaTeX,
             atom.config.get('markdown-preview-plus.useGitHubStyle'),
-            await this.handler.getTeXConfig(),
+            await this.handler.getTeXConfig()
```

Comment:
Pretty sure this trailing comma was here by desing, as formatted by prettier. Contribution guide recommends runnign `npm run prettier`